server/locale/DE/Pi30/mutations/pi30.js

```diff
@@ -0,0 +1,125 @@
+import Logo from '../../../../message/logos';
+import {
+    xSmallFallback,
+    textWrap,
+    messageLogoWidth,
+    altNoWrap,
+    setLogoTop,
+    xSmallNoWrap,
+    addPeriod,
+    primaryWrap
+} from '../../../../message/mediaQueries';
+import { textLogoMutations } from '../../../../message/logoMutations';
+
+export default {
+    'layout:text': [
+        [
+            'default',
+            ({ textSize }) => ({
+                styles: [
+                    textWrap(textSize * 55, textSize, 'DE'),
+                    xSmallFallback(textSize * 13.75),
+                    messageLogoWidth(false, textSize * 4, textSize * 1.25),
+                    setLogoTop(textSize * 20),
+                    xSmallNoWrap(textSize * 7.7),
+                    addPeriod(),
+                    primaryWrap(textSize * 12.4)
+                ],
+                logo: Logo.PP_PAYPAL.COLOR,
+                headline: [
+                    {
+                        tag: 'medium',
+                        br: ['Einkäufen']
+                    },
+                    { tag: 'xsmall', br: [','] }
+                ],
+                disclaimer: ['default']
+            })
+        ],
+        [
+            'logo.type:primary && logo.position:right',
+            ({ textSize }) => ({
+                styles: [
+                    xSmallFallback(textSize * 15),
+                    setLogoTop(textSize * 42),
+                    messageLogoWidth(textSize * 6, textSize * 4, textSize * 1.25),
+                    addPeriod(),
+                    `@media screen and 
+                    (max-width: ${textSize * 29}px) {
+                        .message__headline > .tag--medium > span > span.br:nth-child(2) {white-space: nowrap;}
+                    }`
+                ]
+            })
+        ],
+        [
+            'logo.type:primary && logo.position:top',
+            ({ textSize }) => ({
+                styles: [
+                    messageLogoWidth(textSize * 6, textSize * 4, textSize * 1.25),
```

Comment:
```suggestion
                styles: [
                    xSmallFallback(textSize * 15.1),
                    messageLogoWidth(textSize * 6, textSize * 4, textSize * 1.25),
```

It seems like we were missing the xSmallFallback media query for this style so the fallback was showing in the full message.